guides/earn-grc.htm

```diff
@@ -141,6 +141,18 @@ <h5 class="text-center mt-5">2) Running the Wallet to Stake</h5>
                             </figcaption>
                             o) Keep the wallet up to date as updates are released
                         </div>
+<h5 class="text-center mt-5">3) Monitor Your Crunching</h5>
+                    <div class="mb-5 pb-2">
+                            <p>While you can monitor your work in the BOINC and/or Folding@Home managers, it may be useful to view what work the Gridcoin wallet sees you accomplishing. The Gridcoin wallet will show you how much you've earned, and your <a href="/wiki/magnitude.html">Magnitude</a> by project. To view Magnitude, open the Researcher Configuration screen as described above, and select the Projects tab. Magnitude, as well as Recent Average Credit (RAC) are shown per project. Please note that this updates slowly over time, so it is a good idea to wait a day or two after making changes before drawing conclusions from this information. If you do not see any Magnitude or RAC being populated, or the CPID shown is incorrect, you may have an issue causing you to not receive credit for your work.

```

Comment:
RAC is defined both here and on line 158. I would suggest removing the definition there